pkg/configuration/constants.go

```diff
@@ -28,8 +28,10 @@ const (
 	CACHE_PATH                      string = "snyk_cache_path"
 	TIMEOUT                         string = "snyk_timeout_secs"
 	LOG_LEVEL                       string = "snyk_log_level" // string that defines the log level based on zerolog levels (trace,debug,info,...)
+	CONFIG_FILE                     string = "configfile"     // command line flag from language server
```

Comment:
Issue: Extension specific configuration shouldn't be reflected here. The config value is already registered [here](https://github.com/snyk/snyk-ls/blob/main/ls_extension/main.go#L50) so the question is how to use it without creating circular dependencies. `configfile` as a name is globally seen not very specific. I think I will need some time to have a good solution.

Reply:
removed it for now, so it's back to the status before.